{
  "test_type": "e2e",
  "timestamp": "2025-07-27T19:33:17.968436",
  "bug_id": "mrr_api_misuse_0491",
  "results": {
    "total": 98,
    "passed": 65,
    "failed": 6,
    "skipped": 4
  },
  "failures": [
    {
      "test": "test_api_misuse_0",
      "error": "429 errors",
      "file": "src/utils/common_26.py",
      "line": 177
    },
    {
      "test": "test_api_misuse_1",
      "error": "Requests blocked",
      "file": "src/utils/common_26.py",
      "line": 67
    }
  ]
}